client/src/index.scss

```diff
@@ -6,13 +8,18 @@ html {
   box-sizing: inherit;
 }
 
-body {
+*:focus {
+  outline: none;
+}
+
+h1, h2, h3, h4, h5, h6, body, p, span {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  color: $primary-text;
```

Comment:
If you look at `index.tsx` this was meant to be used as a reset file. so let's not set anything here. It's meant for resetting.